webapp/src/shared/components/Input/Input.tsx

```diff
@@ -0,0 +1,41 @@
+import { Color } from '../../types';
+import Icon, { IconSize, IconVariant } from '../Icon/Icon';
+import './Input.css';
+
+export enum InputColor {
+  LIGHT = 'light',
+  DARK = 'dark',
+}
+
+type InputProps = {
+  color?: InputColor;
+  label?: string;
+  iconVariant?: IconVariant;
+  input?: string;
+};
+
+export default function Input({
+  color,
+  label,
+  iconVariant,
+  input,
```

Comment:
in the component input, everything is input. Maybe `placeholder` would be a name that more accurately would describe what is it that it's actually doing?

Reply:
I stuck to the issue names but you're right, it makes more sense.